packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/index.js

```diff
@@ -233,9 +233,13 @@ const EditViewDataManagerProvider = ({
     ({ target: { name, value, type } }, shouldSetInitialValue = false) => {
       let inputValue = value;
 
-      // Allow to reset text, email, uid, select/enum, and number
+      console.log({ type, name });
```

Comment:
Not sure this one is needed...

Reply:
nice to see you there catching my logs 😁